test/unit/specs/core/createEventManager.spec.js

```diff
@@ -39,7 +39,10 @@ describe("createEventManager", () => {
     consent = jasmine.createSpyObj("consent", {
       awaitConsent: Promise.resolve()
     });
-    event = jasmine.createSpyObj("event", ["setLastChanceCallback"]);
+    event = jasmine.createSpyObj("event", {
+      finalize: undefined,
+      shouldSend: true
+    });
```

Comment:
If you want, you can do `finalize() {}` here and then in your tests below you could use this event rather than creating a new `eventToIgnore` event. To tweak the return value of this event's `shouldSend` method for a specific test, you would do `event.shouldSend.and.returnValue(false)`. Your choice though.